sdk/typescript/dev/src/ci.ts

```diff
@@ -0,0 +1,57 @@
+import { Container, dag, func, object } from "@dagger.io/dagger"
+
+@object
+export class CI {
```

Comment:
I think the engine will namespace this so it'll probably create a TypescriptSdkDevCi object, but I prefer keeping "ci" stuff to the `/ci` (from repo root) module that'll orchestrate all SDKs, engine, CLI... That's what'll go on .workflows.

In this directory, it's about local development, but also making it reusable so that it can be as simple as possible to use from the "ci" module, and should also allow running the linter on a docs snippet, for example, by providing different source files. This way Vikram or Solomon can work on an example, without having node installed locally.

Reply:
I put test, build and lint as top level function but they are scoped to the TypescriptSDK project.
I need to find a good pattern so it can accept standalone typescript files to lint too since docs snippets doesn't have js project in it

Reply:
And I'm not sure it will work without a proper eslint configuration files, I need to do some tests